integration/inspektor-gadget/integration_test.go

```diff
@@ -611,25 +611,23 @@ func TestFsslower(t *testing.T) {
 		Cmd:          fmt.Sprintf("$KUBECTL_GADGET trace fsslower -n %s -f %s -m 0 -o json", ns, fsType),
 		StartAndStop: true,
 		ExpectedOutputFn: func(output string) error {
-			expectedEntries := []*fsslowerType.Event{
-				{
-					Event: BuildBaseEvent(ns),
-					Comm:  "cat",
-					File:  "foo",
-				},
+			expectedEntry := &fsslowerType.Event{
+				Event: BuildBaseEvent(ns),
+				Comm:  "cat",
+				File:  "foo",
+				Op:    "R",
 			}
 
 			normalize := func(e *fsslowerType.Event) {
 				e.Node = ""
 				e.MountNsID = 0
 				e.Pid = 0
-				e.Op = ""
 				e.Bytes = 0
 				e.Offset = 0
 				e.Latency = 0
 			}
 
-			return ExpectEntriesToMatch(output, normalize, expectedEntries...)
+			return ExpectEntriesToMatch(output, normalize, expectedEntry)
```

Comment:
nit: I think you amended this change to the wrong commit. All these changes should be on the first commit of this PR.